internal/config/config.go

```diff
@@ -310,6 +311,144 @@ func setClowderConfiguration(v *viper.Viper, clowderConfig *clowder.AppConfig) {
 	// Metrics configuration
 	v.Set("metrics.path", clowderConfig.MetricsPath)
 	v.Set("metrics.port", clowderConfig.MetricsPort)
+
+	// If TlsCAPath print an informative message
+	if clowderConfig.TlsCAPath != nil {
+		slog.Info("clowder provides a path for CA, be sure it is in your SSL_CERT_DIR",
+			slog.String("TlsCAPath", *clowderConfig.TlsCAPath),
+		)
```

Comment:
Do I read it correctly, that if the clowder config includes the `TlsCAPath`, then we want to advise the user that `SSL_CERT_DIR` env var should be set, and should include that path?

If that's correct, perhaps we could consider to read and parse `SSL_CERT_DIR` and ensure it - and die (or log a much more serious error level) if we detect a misconfiguration.